StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 1, 2 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 2, 3 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 3, 4 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 4, 5 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 5, 6 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 6, 7 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 7, 8 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 8, 9 )

StudentTest1.cpp:166: ERROR: CHECK_EQ( team2.stillAlive(), i + 2 ) is NOT correct!
  values: CHECK_EQ( 9, 10 )

StudentTest1.cpp:171: ERROR: CHECK_THROWS_AS( team1.add(over), std::runtime_error ) did NOT throw at all!
